LICENSE.md

```diff
@@ -1 +0,0 @@
-This software is dual-licensed under [Apache 2.0 license](LICENSE-Apache-2.0.txt) OR [GNU LESSER GENERAL PUBLIC LICENSE version 2.1](LICENSE.LGPLv21.txt).
```

Comment:
I don't know it is right remove this file. What do you think @bettio ?

Reply:
I think this should be replaced by the LICENSES folder.

Reply:
Sadly GitHub doesn't detect yet the LICENSES directory, so we would loose the License field in the project summary.
So, let's replace this with the Apache-2.0 license file.